ui/pages/confirmations/components/confirm/info/approve/spending-cap/spending-cap.tsx

```diff
@@ -91,9 +91,9 @@ export const SpendingCap = ({
   );
 
   const accountBalance = calcTokenAmount(
-    userBalance || '0',
-    Number(decimals || '0'),
-  );
+    userBalance ?? '0',
+    decimals,
```

Comment:
```suggestion
    decimals ?? '0',
```
calcTokenAmount does not currently handle if decimals == null. Not sure if we should be concerned but maybe just in case?